spec/services/container_dashboard_service_spec.rb

```diff
@@ -25,4 +26,74 @@
                                     }])
     end
   end
+
+  context "node_utilization" do
+    it "show aggregated metrics from last 30 days only" do
+      MiqRegion.seed
+      @zone = EvmSpecHelper.create_guid_miq_server_zone[2]
+      @time_profile = FactoryGirl.create(:time_profile_utc)
+      ems_openshift = FactoryGirl.create(:ems_openshift, :zone => @zone)
+      ems_kubernetes = FactoryGirl.create(:ems_kubernetes, :zone => @zone)
+
+      current_date = 7.days.ago
+      old_date = 35.days.ago
+
+      current_metric = FactoryGirl.create(
+        :metric_rollup_cm_daily,
+        :timestamp                => current_date,
+        :derived_memory_used      => 1024,
+        :derived_vm_numvcpus      => 2,
+        :derived_memory_available => 2048,
+        :cpu_usage_rate_average   => 100,
+        :time_profile             => @time_profile)
+
+      old_metric = FactoryGirl.create(
+        :metric_rollup_cm_daily,
+        :timestamp                => old_date,
+        :derived_memory_used      => 1024,
+        :derived_vm_numvcpus      => 2,
+        :derived_memory_available => 2048,
+        :cpu_usage_rate_average   => 100,
+        :time_profile             => @time_profile)
+
+      ems_openshift.metric_rollups << current_metric
+      ems_openshift.metric_rollups << old_metric
+      ems_kubernetes.metric_rollups << current_metric.dup
+      ems_kubernetes.metric_rollups << old_metric.dup
```

Comment:
@zeari I think that to be on the safe-side you should use different values for the two providers (e.g. 4 cpus vs 2, 4096 avail mem vs 2048, etc.). Because at the moment a bug that aggregates twice just one of the two providers would pass the `node_utilization_all_providers` test.